paimon-common/src/main/java/org/apache/paimon/CoreOptions.java

```diff
@@ -219,6 +219,12 @@ public class CoreOptions implements Serializable {
                     .defaultValue(Duration.ofSeconds(10))
                     .withDescription("The discovery interval of continuous reading.");
 
+    public static final ConfigOption<Integer> SCAN_SPLIT_MAX_PER_TASK =
+            key("scan.split-max-per-task")
```

Comment:
`scan.max-splits-per-task`?
And the description should be more...

Reply:
done